src/common/adapters/jira.js

```diff
@@ -20,18 +20,22 @@ const adapter = {
   inspect(loc, doc, fn) {
     if (doc.body.id !== 'jira') return fn(null, null);
 
-    if ($has('.ghx-fieldname-issuekey a', doc)) { // JIRA sidebar
-      const id = $text('.ghx-fieldname-issuekey a', doc);
-      const title = $text('[data-field-id="summary"]', doc);
-      const type = normalizeType($attr(`[data-issue-key="${id}"] .ghx-type`, doc, 'title'));
+    if ($has('.ghx-backlog-column .ghx-backlog-card.ghx-selected', doc)) {
```

Comment:
Just a quick look at the DOM in a Jira backlog and there seems to be an additional `ghx-selected-primary` class which is only used **once** (vs. one can select multiple tickets which will all have the `ghx-selected` class).

How does the current approach handle multiple tickets being selected? I assume it just take the very first one 🤔

Reply:
good question. I'll investiage ⚙️